dev-vcs/tig/tig-2.2.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A text mode interface for git"
+DESCRIPTION="
+Tig functions mainly as a git repository browser. It can also help in the \
+staging changes for commit at chunk level and act as a pager for output \
+different git commands."
+HOMEPAGE="http://jonas.nitro.dk/tig/"
+SOURCE_URI="http://jonas.nitro.dk/tig/releases/tig-2.2.tar.gz"
```

Comment:
Could you please change this to:
```
SOURCE_URI="http://jonas.nitro.dk/tig/releases/tig-$portVersion.tar.gz"
```

While at it, could you please sort this hole section to have the parameters in this order:
```
SUMMARY
DESCRIPTION
HOMEPAGE
COPYRIGHT
LICENSE
REVISION
SOURCE_URI
CHECKSUM_SHA256
```